src/main/resources/liquibase/202207061021-migrate-cores-sockets-data.xml

```diff
@@ -0,0 +1,59 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<databaseChangeLog
+  xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+  xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog
+        http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.1.xsd">
+
+  <changeSet id="202207061021-1" author="khowell" dbms="postgresql">
+    <comment>Copy data from hosts.cores to instance_measurements.</comment>
+    <sql>
+      insert into instance_measurements(instance_id, uom, value)
+      select instance_id::uuid, 'CORES', cores
+      from hosts
+      where cores is not null on conflict(instance_id, uom) do
+      update set value =excluded.value;
+    </sql>
+  </changeSet>
+  <changeSet id="202207061021-2" author="khowell" dbms="postgresql">
+    <comment>Copy data from hosts.sockets to instance_measurements.</comment>
+    <sql>
+      insert into instance_measurements(instance_id, uom, value)
+      select instance_id::uuid, 'SOCKETS', sockets
+      from hosts
+      where sockets is not null
+      on conflict(instance_id, uom) do update set value=excluded.value;
+    </sql>
+  </changeSet>
+  <changeSet id="202207061021-3" author="khowell" dbms="postgresql">
+    <comment>Copy data from hardware_measurements.sockets to tally_measurements.</comment>
+    <sql>
+      insert into tally_measurements(snapshot_id, measurement_type, uom, value)
+      select snapshot_id, measurement_type, 'SOCKETS', sockets
+      from hardware_measurements
+      where sockets is not null
+      on conflict(snapshot_id, measurement_type, uom) do update set value=excluded.value;
+    </sql>
+  </changeSet>
+  <changeSet id="202207061021-4" author="khowell" dbms="postgresql">
+    <comment>Copy data from hardware_measurements.cores to tally_measurements.</comment>
+    <sql>
+      insert into tally_measurements(snapshot_id, measurement_type, uom, value)
+      select snapshot_id, measurement_type, 'CORES', cores
+      from hardware_measurements
+      where cores is not null
+      on conflict(snapshot_id, measurement_type, uom) do update set value=excluded.value;
+    </sql>
+  </changeSet>
+  <changeSet id="202207061021-5" author="khowell" dbms="postgresql">
+    <comment>Copy data from hardware_measurements.cores to tally_measurements.</comment>
```

Comment:
Copy paste error in the comment here I think?
```suggestion
    <comment>Copy data from hardware_measurements.instance_count to tally_measurements.</comment>
```

Reply:
@awood fixed, PTAL